uint32 pwm_value_1
uint32 pwm_value_2
string direction_1
string direction_2
